pkg/fb/fb.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fb
+
+import (
+	"fmt"
+	"image"
+	"io/ioutil"
+	"os"
+
+	framebuffer "github.com/orangecms/go-framebuffer"
```

Comment:
is there any way you can make the dirname / reponame == go package name?

perhaps put the go stuff in github.com/orangecms/go-framebuffer/framebuffer? or /fb?

Reply:
Yea that's a good idea! So that's to get rid of the alias, right? Or would it solve any other issue I am missing?

Reply:
done